policy/resolver.go

```diff
@@ -585,6 +585,11 @@ func (s *LocalServices) tryResolve(ctx context.Context, bundleMrn string, assetF
 	reportingJobUUIDs := topologicalSortReportingJobs(collectorJob.ReportingJobs)
 	for _, rjUUID := range reportingJobUUIDs {
 		rj := collectorJob.ReportingJobs[rjUUID]
+		if rj.QrId == "root" {
```

Comment:
can this also be fixed by moving the pruning to after the frameworks also add their RJs?

Reply:
i'm not sure. We always report frameworks. I don't know if that is an accident or explicitly wanted. If we move it, those get pruned